src/main/resources/application-replica.yaml

```diff
@@ -0,0 +1,2 @@
+primary:
+  publicationUrl: postgresql://${POSTGRES_CLUSTER_HOST:192.168.49.2}:${POSTGRES_PORT:5432}
```

Comment:
Is there a way we can not have this available at all in prod?  I.e. a local dev profile only?  The whole idea about putting this in it's own file is to avoid it's usage everywhere except locally.  As I understand it the `replica` profile is what is run everywhere so this will effectively be wired in everywhere when it is not required.

Reply:
Maybe you needed it everywhere?

Reply:
replicaSubscriptionUrl

Reply:
Renamed.